src/test/java/no/unit/nva/bare/AuthorityProxyTest.java

```diff
@@ -0,0 +1,213 @@
+package no.unit.nva.bare;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.google.gson.reflect.TypeToken;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.lang.reflect.Type;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class AuthorityProxyTest {
+
+    @Rule
+    public MockitoRule rule = MockitoJUnit.rule();
+
+    @Mock
+    BareConnection mockBareConnection;
+
+    @Before
+    public void setUp() {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    @Test
+    public void testSuccessfulResponseWithNameParam() throws IOException {
+        AuthorityProxy mockAuthorityProxy = new AuthorityProxy(mockBareConnection);
+        InputStream inputStream = AuthorityProxyTest.class.getResourceAsStream("/bareResponse.json");
+        InputStreamReader bareResponseStreamReader = new InputStreamReader(inputStream);
+        when(mockBareConnection.connect(any())).thenReturn(bareResponseStreamReader);
+        when(mockBareConnection.generateQueryUrl(anyString())).thenCallRealMethod();
+        String postRequestBody = "{\n"
```

Comment:
I prefer the data to be in files and the intention of the file content to be noted as a constant